package.json

```diff
@@ -71,6 +71,7 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.24",
     "@fortawesome/free-solid-svg-icons": "^5.11.1",
     "@fortawesome/react-fontawesome": "^0.1.4",
+    "@mapeo/core": "^8.1.3",
```

Comment:
this is for the errors, which makes me think even more that all of that code in src/main/mapeo.js ought to instead be in mapeo/core itself.